modules/dxp/apps/commerce/commerce-organization-web/src/main/resources/META-INF/resources/js/D3OrganizationChart.js

```diff
@@ -30,13 +30,21 @@ import {
 	getEntityId,
 	getMinWidth,
 	hideChildren,
+	insertAddButtons,
 	insertChildrenIntoNode,
 	showChildren,
 	tree,
 } from './utils/index';
-import {fillEntityNode, getLinkDiagonal} from './utils/paint';
+import {fillAddButtons, fillEntityNode, getLinkDiagonal} from './utils/paint';
```

Comment:
Could you add a new line between L37 and L38?

Reply:
Unfortunately it throws: `imports must be grouped (unexpected blank line before: "./utils/paint")`

Reply:
> Could you add a new line between L37 and L38?

@diegonvs: Always trust the formatter :wink: My point being, if SF passes we shouldn't try to change the output and it's also confusing for other teams who send a PR having seen SF pass locally get a comment like this. *Of course, if you really think we should change this, let's talk it with the rest of @liferay-frontend in Slack or GitHub.*